code/__defines/skills.dm

```diff
@@ -12,25 +12,14 @@
 #define SKILL_AVERAGE  2
 #define SKILL_HARD     4
 
-#define SKILL_BUREAUCRACY   /decl/hierarchy/skill/organizational/bureaucracy
-#define SKILL_FINANCE       /decl/hierarchy/skill/organizational/finance
 #define SKILL_EVA           /decl/hierarchy/skill/general/EVA
-#define SKILL_MECH          /decl/hierarchy/skill/general/mech
-#define SKILL_PILOT         /decl/hierarchy/skill/general/pilot
 #define SKILL_HAULING       /decl/hierarchy/skill/general/hauling
+#define SKILL_PILOT         /decl/hierarchy/skill/general/pilot
 #define SKILL_COMPUTER      /decl/hierarchy/skill/general/computer
-#define SKILL_BOTANY        /decl/hierarchy/skill/service/botany
 #define SKILL_COOKING       /decl/hierarchy/skill/service/cooking
 #define SKILL_COMBAT        /decl/hierarchy/skill/security/combat
 #define SKILL_WEAPONS       /decl/hierarchy/skill/security/weapons
-#define SKILL_FORENSICS     /decl/hierarchy/skill/security/forensics
 #define SKILL_CONSTRUCTION  /decl/hierarchy/skill/engineering/construction
```

Comment:
Request construction be split up into two or more granular skills, not entirely certain which yet, it needs thought

Reply:
I was thinking to add a 'Defensive Construction' skill, lorewise something a combat engineer or sapper might pick up, which would allow access to defensive structures like traps.